cli_reference/get_started_cli.adoc

```diff
@@ -135,9 +178,16 @@ Now using project 'project03'.
 $ osc project
 Using project 'project03'.
 ----
```

Comment:
There's a lot going on in this paragraph, and we generally show the commands in the sidebar block, not as part of the intro text. I think we should separate this into the different tasks. For example:

Use the following command to view the project you are currently using:
`osc project`

Switch to a particular project by specifying the name of the project:
`osc project _<project>_`